go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	golang.org/x/tools v0.0.0-20200723000907-a7c6fd066f6d // indirect
 	google.golang.org/genproto v0.0.0-20200117163144-32f20d992d24 // indirect
 	gopkg.in/yaml.v2 v2.3.0
+	gotest.tools v2.2.0+incompatible
```

Comment:
looks like an unintended addition of a dependency.   The cause is noted below with `resolver_incluster_test.go`